core/vm/eips.go

```diff
@@ -83,8 +87,20 @@ func opSelfBalance(pc *uint64, interpreter *EVMInterpreter, callContext *callCtx
 	return nil, nil
 }
 
+// enable2937 applies EIP-2937 (SET_INDESRUCTIBLE Opcode)
+// - Adds an opcode that prevents contract from calling SELFDESTRUCT (0xFF)
+func enable2937(jt *JumpTable) {
+	// New opcode
+	jt[SETINDESTRUCTIBLE] = &operation{
+		execute:     opSetIndestructible,
+		constantGas: GasQuickStep,
+		minStack:    minStack(0, 1),
+		maxStack:    maxStack(0, 1),
+	}
+}
+
 // enable1344 applies EIP-1344 (ChainID Opcode)
-// - Adds an opcode that returns the current chain’s EIP-155 unique identifier
+// - Adds an opcode that returns the current chain's EIP-155 unique identifier
```

Comment:
Generally, the geth team prefers PRs which modify only code which pertain to the work at hand. I recommend reverting this change and apply in a separate PR upstream.